client/src/pages/course/admin/stage-interviews.tsx

```diff
@@ -60,7 +60,13 @@ function Page() {
       </Row>
 
       <Table
-        pagination={{ pageSize: 50 }}
+        pagination={{
+          defaultPageSize: 50,
+          showSizeChanger: true,
+          showTotal: (total, range) => {
+            return `${range[0]}-${range[1]} of ${total} interviews`;
+          },
```

Comment:
Minor but I would inline:

```suggestion
          showTotal: (total, range) => `${range[0]}-${range[1]} of ${total} interviews`,
```

Reply:
nop, fixed in 9a1260b